packages/mesos/buildinfo.json

```diff
@@ -1,24 +1,29 @@
 {
-  "requires": ["openssl", "libevent", "curl", "boost-libs"],
-  "single_source" : {
+  "requires": [
+    "openssl",
+    "libevent",
+    "curl",
+    "boost-libs"
+  ],
+  "single_source": {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "97de6eea9ad70f700082f70cd8d2584e199186df",
-    "ref_origin" : "dcos-mesos-master-e91ce42ed"
+    "ref": "7659c405f51c1771fd341c9fad34085982774014",
```

Comment:
Ack that this is the tip of the `dcos-mesos-master-nightly-a1c6a7a` branch.
How can I verify that this is the desired branch though?
Would an encoding of the date after the word "nightly" make more sense than a random sha?

Reply:
The SHA helps us identify the appropriate apache/mesos commit on top of which we applied the cherry-picks. We can certainly add some date as well but the SHA should still be left in there.

Reply:
I see, so the `a1c6a7a` in `dcos-mesos-master-nightly-a1c6a7a` is the SHA on upstream master on top of which all of the commits below are being cherry-picked. No need for a date then, that's clear enough.

Reply:
Yup.